docs/stories/stories.md

```diff
@@ -0,0 +1,79 @@
+- Confirmation request emails will tell the user to modify the text if incorrect
+- Attribution authoriser is notified of a new attribution at their institution
+- Attribution authoriser can mark an attribution confirmation email as valid
+- Attribution authoriser can see highlighted changes to attribution confirmation emails (compared to original message)
+
+### Project approval and system allocation
+- Project approvers, RSEs and attribution authorisers can see a page listing pending/unapproved projects per institution
+- Project approvers, RSEs and attribution authorisers can see details of pending/unapproved projects per institution
+- Project approvers can set project status to approved
+- Project approvers can reject project with a comment (project owner receives comment)
+- Project approvers can specify a system (or systems) on which a system allocation is created
+- Project approvers can modify default values of that system allocation
+- Project approvers can add new system allocations (for example on another cluster system or over a new time period) to an existing project
+- System allocations allowable and their priority should be determined in some way by level and/or nature of attribution (policy to be agreed by management board)
+- The associated cluster user accounts are enabled on the appropriate cluster when new (active) system allocations are created
+- Historical comments on a project are stored in a separate table
+ 
+### Cluster accounts
+- Can users have a cluster accounts on a cluster system (or multiple systems)
+- System administrators can create cluster user accounts for users
+- Creating a cluster account which belongs to a system will create an account on the appropriate cluster
+- Users should be able to reset passwords for their cluster accounts, user passwords will not be stored on the cogs system
+ 
+### Raven users
+- New raven users will automatically have a new cluster account set up on first login and will belong to a default "Raven" project
+- Active cluster user accounts created on the Cardiff system with no approval required
+ 
+### Potential future changes
+- Attribution should automatically affect resource allocation by some mechanism (according to policy agreed by management board)
+- All database records should contain a created and modified time stamp for auditing purposes. Records likely to be subject to auditing (e.g. approvals) should be versioned as appropriate
+ 
+### Other Suggestions
+- Projects who aren't returning anything (yet) need a grant number to be raised from bottom category
+- Storage is allocated in system allocation table
+- On boarding and training could be combined to a single field
+- Paragraph about attribution from previous SCW form should be included
```

Comment:
We can add a text field for attributions now.  Technical Lead could then edit/add to.

Reply:
Added to project board.

Reply:
There is a specific block of text that has been written that was requested to go in (at least for Swansea). I can get hole of that and add it.